common/artifacts/000_wc_artifacts.txt

```diff
@@ -838,6 +1314,154 @@ prosthetic_blade_hand = {
 
 	picture = "GFX_inv_weapon_shortblade_29"
 }
+warsong_battle_drums = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes
+		
+	}
+
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+	command_modifier = {
+		heavy_infantry = 0.05
+		light_infantry = 0.05
```

Comment:
`light_infantry` gives bonus to Archers and Light Infantry. `actual_light_infantry` to Light Infantry only. Check how vanilla artifacts work.
```suggestion
		actual_light_infantry = 0.05
```

Reply:
done